core/src/main/java/org/apache/accumulo/core/client/admin/servers/ServerTypeName.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.client.admin.servers;
+
+/**
+ * Server process type names that a client can be expected to interact with. Clients are not
+ * expected to interact directly with the GarbageCollector or Monitor processes.
+ *
+ * @since 4.0.0
+ */
+public enum ServerTypeName {
```

Comment:
I think "Name" is already implied. This should probably just be `ServerType`

Reply:
This enum is strongly related to ServerId.  Could name it ServerIdType or move the enum under ServerId and name it Type, when when using it could have ServerId.Type.

Reply:
It's now named ServerId.Type.